src/utils/helper-functions/guards.ts

```diff
@@ -1,9 +1,10 @@
 import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/experimental-utils'
 import ts from 'typescript'
 
-const isNodeOfType = <NodeType extends AST_NODE_TYPES>(nodeType: NodeType) => (
-  node: TSESTree.Node,
-): node is TSESTree.Node & { type: NodeType } => node.type === nodeType
+const isNodeOfType =
+  <NodeType extends AST_NODE_TYPES>(nodeType: NodeType) =>
+  (node: TSESTree.Node): node is TSESTree.Node & { type: NodeType } =>
+    node.type === nodeType
```

Comment:
I was seeing the following report by `prettier` (here and in /use-effects-lifecycle-interface.ts):

<img width="671" alt="Prettier report" src="https://user-images.githubusercontent.com/11965907/131406571-e856b656-6813-4919-86de-9b89e83f6438.png">